### PR TITLE
[zh-cn] create the translation of "Test your skills: Conditionals"

### PR DESCRIPTION
**Pull request by TripleCamera**
### Description

Translate `Learn/JavaScript/Building_blocks/Test_your_skills:_Conditionals`.

This is my first pull request. If there are problems in my translation, please point them out.

### Motivation

:blush:

### Additional details

Source: [Test your skills: Conditionals](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Test_your_skills:_Conditionals)

[Roy-tian's GitHub repo](https://github.com/roy-tian/learning-area) needs update!

Should we create templates for the "use online editor and reach out for help" note and the "download the starting point" callout which appear rapidly in "test your skills"? That would be more convenient.

### CODE REVIEW

File: files/zh-cn/learn/javascript/building_blocks/test_your_skills_colon__conditionals/index.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+> [!CALLOUT]
+>
+> [下载此任务的初始代码](https://github.com/mdn/learning-area/blob/main/javascript/building-blocks/tasks/conditionals/conditionals2-download.html)以在自己的编辑器或在线编辑器中完成任务。
+
+## 条件语句 3
+
+在最后一项任务中，你有四个变量：
+
+- `machineActive`——包含一个指示器（布尔值），指示登录机器是否启动（`true`/`false`）。
```
Comment (yin1999) [inline files/zh-cn/learn/javascript/building_blocks/test_your_skills_colon__conditionals/index.md:67]:
```suggestion
- `machineActive`——包含一个指示器，指示登录机器是否启动（`true`/`false`）。
```

File: files/zh-cn/learn/javascript/building_blocks/test_your_skills_colon__conditionals/index.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+---
+title: 技能测试：条件语句
+slug: Learn/JavaScript/Building_blocks/Test_your_skills:_Conditionals
+l10n:
+  sourceCommit: c0f1aecaed48d75652c6dd97f30c7febd07e5cde
```
Comment (yin1999) [inline files/zh-cn/learn/javascript/building_blocks/test_your_skills_colon__conditionals/index.md:5]: 对应的英文文档的最新提交的 commit sha

```suggestion
  sourceCommit: bc0d0d1ef796435e969f6d65c7e5d3c08f4023aa
```

**Comment by yin1999** [comment]
> Should we create templates for the "use online editor and reach out for help" note and the "download the starting point" callout which appear rapidly in "test your skills"? That would be more convenient.

不同的文档用了不同的备注，难以创建统一的模板，同时也不容易维护。

**Comment by TripleCamera** [comment]
> 不需要用转义字符

好的。一开始我用了转义字符是因为这些各种各样的横杠用肉眼难以区分 :thinking:

**Review by yin1999 — APPROVED** [review]
感谢 @TripleCamera :+1:

**Pull request merged**